protobuf fails to build with too much parallelization

When running `make -j64`, `protobuf` initially fails to build.  After some retries, it works.  So I suspect there is a missing dependency somewhere?

```
Failed to build package protobuf for target x86_64-pc-linux-gnu!
------------------------------------------------------------
configure.ac:25: error: possibly undefined macro: AC_PROG_LIBTOOL
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
autoreconf-2.69: /usr/bin/autoconf-2.69 failed with exit status: 1
make[1]: *** [Makefile:674: build-only-protobuf_x86_64-pc-linux-gnu] Error 1
make[1]: Leaving directory '/tmp/mxe'
real    0m26.555s
user    0m21.740s
sys     0m0.612s
------------------------------------------------------------
[log]      /tmp/mxe/log/protobuf_x86_64-pc-linux-gnu
```

--- a/plugins/native/libtool.mk
+++ b/plugins/native/libtool.mk
@@ -1,7 +1,6 @@
 # This file is part of MXE. See LICENSE.md for licensing information.
 
 PKG                  := libtool
-$(PKG)_TARGETS       := $(BUILD)
 $(PKG)_DEPS_$(BUILD) := autoconf automake
 
 define $(PKG)_BUILD_$(BUILD)

--- a/src/libtool.mk
+++ b/src/libtool.mk
@@ -1,5 +1,6 @@
 # This file is part of MXE. See LICENSE.md for licensing information.
 
+# this pkg is the base for both src/libltdl and plugins/native/libtool
 PKG             := libtool
 $(PKG)_WEBSITE  := https://www.gnu.org/software/libtool/
 $(PKG)_DESCR    := GNU Libtool
@@ -10,6 +11,7 @@ $(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
 $(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
 $(PKG)_URL      := https://ftp.gnu.org/gnu/$(PKG)/$($(PKG)_FILE)
 $(PKG)_DEPS     :=
+$(PKG)_TARGETS  := $(BUILD)
 
 define $(PKG)_UPDATE
     $(WGET) -q -O- 'https://ftp.gnu.org/gnu/libtool/?C=M;O=D' | \

--- a/src/pkgconf.mk
+++ b/src/pkgconf.mk
@@ -10,6 +10,7 @@ $(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
 $(PKG)_URL      := https://github.com/$(PKG)/$(PKG)/tarball/$($(PKG)_VERSION)/$($(PKG)_FILE)
 $(PKG)_TARGETS  := $(BUILD) $(MXE_TARGETS)
 $(PKG)_DEPS     :=
+$(PKG)_DEPS_$(BUILD) := libtool
 
 $(PKG)_UPDATE    = $(call MXE_GET_GITHUB_SHA, pkgconf/pkgconf, master)
 

--- a/src/protobuf.mk
+++ b/src/protobuf.mk
@@ -10,7 +10,7 @@ $(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
 $(PKG)_URL      := https://github.com/google/$(PKG)/archive/v$($(PKG)_VERSION).tar.gz
 $(PKG)_DEPS     := gcc googlemock googletest zlib
 $(PKG)_TARGETS  := $(BUILD) $(MXE_TARGETS)
-$(PKG)_DEPS_$(BUILD) := googlemock googletest
+$(PKG)_DEPS_$(BUILD) := googlemock googletest libtool
 
 define $(PKG)_UPDATE
     $(call MXE_GET_GITHUB_TAGS, google/protobuf, v)

